deliver/lib/deliver/options.rb

```diff
@@ -114,6 +114,11 @@ def self.available_options
                                      description: "Submit the new version for Review after uploading everything",
                                      is_string: false,
                                      default_value: false),
+        FastlaneCore::ConfigItem.new(key: :reject_if_waiting_for_review,
+                                     env_name: "DELIVER_REJECT_IF_WAITING_FOR_REVIEW",
```

Comment:
@joshdholtz @KrauseFx  Do you think we should change the option name from `reject_if_waiting_for_review` to `reject_version_if_possible`? The app can be rejected in 3 states, 'waiting for review', 'in review' and 'Pending Release'. Naming it `reject_if_waiting_for_review` would be misleading to other devs.

Reply:
I think this makes sense ^